docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -64,54 +72,73 @@ $ sudo snap set edgexfoundry support-notifications=off
+The EdgeX snaps support configuration overrides via snap configure hooks which generate service-specific .env files which are used to
+provide a custom environment to the service, overriding the default configuration provided by the service's ```configuration.toml```
+file. If a configuration override is made after a service has already started, then the service must be **restarted** via command-line
+(e.g. ```snap restart edgexfoundry.<service>```), snapd's REST API, or the SMA (sys-mgmt-agent). If the overrides are provided via the
+snap configuration defaults capability of a gadget snap, the overrides will be picked up when the services are first started.
 
-All default configuration files are shipped with the snap inside $SNAP/config, however because $SNAP isn't writable, all of the config files are copied during snap installation (specifically during the install hook, see snap/hooks/install in this repository) to $SNAP_DATA/config.
+The following syntax is used to specify service-specific configuration overrides for the edgexfoundry snap:
 
-```
-$ sudo snap restart edgexfoundry
-```
-## Viewing logs
+```env.<service>.<stanza>.<config option>```
 
-Currently, all log files for the snap's can be found inside $SNAP_COMMON, which is usually /var/snap/edgexfoundry/common. Once all the services are supported as daemons, you can also use sudo snap logs edgexfoundry to view logs.
+For instance, to setup an override of Core Data's Port use:
 
-Additionally, logs can be viewed using the system journal or snap logs. To view the logs for all services in the edgexfoundry snap use:
+```$ sudo snap set edgexfoundry env.core-data.service.port=2112```
 
-```
+And restart the service:
+
+```$ sudo snap restart edgexfoundry.core-data```
+
+**Note** - at this time changes to configuration values in the [Writable] section are not supported.
+
+For details on the mapping of configuration options to Config options, please refer to [Service Environment Configuration Overrides](https://github.com/edgexfoundry/edgex-go/blob/main/snap/README.md#configuration-overrides). For details on configuration overrides for App Service Configurable or device service snaps, please refer to the snap documentation for each. Ex.
+
+`https://github.com/edgexfoundry/device-camera-go/blob/main/snap/README.md`
+
+## Viewing logs
+To view the logs for all services in an EdgeX snap use the `snap log` command:
+
+```bash
 $ sudo snap logs edgexfoundry
 ```
+
 Individual service logs may be viewed by specifying the service name:
 
-```
+```bash
 $ sudo snap logs edgexfoundry.consul
 ```
-Or by using the systemd unit name and journalctl:
 
-```
+Or by using the systemd unit name and `journalctl`:
+
+```bash
 $ journalctl -u snap.edgexfoundry.consul
 ```
-## Security services
 
-Currently, the security services are enabled by default. The security services consitute the following components:
+These techniques all snaps including application and device services.
```

Comment:
`These techniques all snaps ` ?? Not sure what this means

Reply:
fixed